src/pages/auth/login.tsx

```diff
@@ -18,7 +18,7 @@ const LoginPage = (): JSX.Element => {
   const handleSubmit = (e: React.ChangeEvent<HTMLFormElement>): void => {
     e.preventDefault()
 
-    signIn('credentials', { email, password, callbackUrl: 'http://localhost:3000/protected', redirect: false })
```

Comment:
@ThePickleGawd generally never put `localhost:3000` in your code; it is unnecessary and you don't want your code to depend on the port that next is using.